rebar.config

```diff
@@ -1,6 +1,6 @@
 {erl_opts, [warnings_as_errors,
             warn_export_all,
-            {lager_truncation_size, 8192}]}.
+            {lager_truncation_size, 15360}]}.   %% 15KB
```

Comment:
does this mean trucate after 15KiB in one log line?

Reply:
yeah.